src/main/java/de/tum/in/www1/bamboo/server/ServerNotificationTransport.java

```diff
@@ -267,7 +287,7 @@ private JSONObject createJSONObject(Notification notification) {
                     // TODO: This ensures outdated versions of Artemis can still process the new request. Will be removed without further notice in the future
                     buildDetails.put("failedJobs", jobs);
                 }
-
+                buildDetails.put("artifact", hasArtifacts);
```

Comment:
please double check with the Artemis way of showing build results (see Test exercise CI Test)

Reply:
the logic should be as follows: if there is one "classical" Artemis artifact, this should be true, otherwise false.
This means, that static analysis artifacts should NOT be considered here: if there is a spot bugs artifact, but no "classical" Artemis artifact, then this property should be false!

Reply:
also make sure on the Artemis side that in the method BambooService.retrieveLatestArtifact(...) all static code analysis artifacts are filtered so that the actual "classical" Artemis artifact is downloaded

Reply:
Reverted how the artifact flag is set. The old way only checks for shared artifacts (contained in the resultsSummary). We won't configure static assessment artifacts as shared as there is no good reason for it. This means static assessment artifacts won't be considered for setting the flag.

Reply:
If both static assessment and classical artifacts are present, BambooService.retrieveLatestArtifact will pick the first artifact in a list and provide it as a download. Filtering of static assessment artifacts and other bugfixes (e.g. don't show download button for template and solution participation) will be implemented as another PR for Artemis.